### PR TITLE
[Teams] browser isolation policies

### PR DESCRIPTION
**Pull request by abracchi-tw**

### CODE REVIEW

File: products/cloudflare-one/src/content/policies/browser-isolation/index.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+---
+order: 4
+pcx-content-type: concept
+---
+
+# Browser Isolation
+
+Just like how you can use Gateway to allow or block traffic based on content categories or security threats, you can define Isolation policies to dynamically isolate websites based on identity, security threats or content. To build Browser Isolation policies, navigate to **Policies > HTTP policies** on the Teams Dashboard. In the rule builder, choose the *Isolate* or *Do not Isolate* actions to enable or disable isolation for certain websites or content.
```
Comment (marciocloudflare) [inline products/cloudflare-one/src/content/policies/browser-isolation/index.md]:
```suggestion
Just like how you can use Gateway to allow or block traffic based on content categories or security threats, you can define Isolation policies to dynamically isolate websites based on identity, security threats or content. To build Browser Isolation policies, navigate to **Policies** > **HTTP policies** on the Teams Dashboard. In the rule builder, choose the *Isolate* or *Do not Isolate* actions to enable or disable isolation for certain websites or content.
```

File: products/cloudflare-one/src/content/policies/browser-isolation/index.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+![Browser isolation policy](../../static/documentation/policies/bi-policy.png)
+
+## Isolate
+
+When an HTTP policy applies the Isolate action, the user's web browser is transparently served a HTML compatible remote browser client. Isolation policies can be applied to requests that include `Accept: text/html*`. This allows Browser Isolation policies to co-exist with API traffic.
```
Comment (marciocloudflare) [inline products/cloudflare-one/src/content/policies/browser-isolation/index.md]:
```suggestion
When an HTTP policy applies the Isolate action, the user's web browser is transparently served an HTML compatible remote browser client. Isolation policies can be applied to requests that include `Accept: text/html*`. This allows Browser Isolation policies to co-exist with API traffic.
```

File: products/cloudflare-one/src/content/policies/browser-isolation/index.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+| Selector | Operator | Value | Action |
+| - | - | - | - |
+| Host | In | `example.com`, `example.net` | Isolate
+
+<Aside type='note' header='Isolate identity providers for applications'>
+
+Existing cookies and sessions from non-isolated browsing are not sent to the remote browser. Websites that implement single sign on using third-party cookies will also need to be isolated.
+
+For example, example.com authenticates using Google Workspace you will also need to isolate the top level <a href="https://support.google.com/a/answer/9012184">Google Workspace URLs</a>.
```
Comment (marciocloudflare) [inline products/cloudflare-one/src/content/policies/browser-isolation/index.md]: I think there's something missing in this sentence after `Google Workspace`.

Also I think example.com should be between backticks. Wdyt?

File: products/cloudflare-one/src/content/policies/browser-isolation/index.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+| Selector | Operator | Value | Action |
+| - | - | - | - |
+| Host | In | `example.com` | Do Not Isolate |
+
+## Settings
+
+Malware and zero-day threats are not the only security challenges administrators face with web browsers. The mass adoption of SaaS products has made the web browser the primary tool used to access data. Lack of control over both the application and the browser has left administrators little control over their data once it is delivered to an endpoint.
+
+All following settings can be applied to websites through Applications, Lists, Domain and Hostname expressions.
```
Comment (marciocloudflare) [inline products/cloudflare-one/src/content/policies/browser-isolation/index.md]:
```suggestion
All the following settings can be applied to websites through Applications, Lists, Domain and Hostname expressions.
```

Comment (marciocloudflare) [reply products/cloudflare-one/src/content/policies/browser-isolation/index.md]: Not sure but I think this sounds better this way. Wdyt?

**Review by marciocloudflare — APPROVED** [review]

**Review by Dereck1980 — APPROVED** [review]

File: products/cloudflare-one/src/content/policies/filtering/http-policies/index.md [hunk 5]
```diff
@@ -54,35 +54,11 @@ Rules with Block actions block outbound traffic from reaching destinations you s
 
 ### Isolate
```
Comment (Dereck1980) [inline products/cloudflare-one/src/content/policies/filtering/http-policies/index.md:56]: I want it

**Review by Dereck1980 — APPROVED** [review]

File: products/cloudflare-one/src/content/policies/filtering/http-policies/index.md (hunk 5 above)
Comment (Dereck1980) [reply products/cloudflare-one/src/content/policies/filtering/http-policies/index.md:56]: It all good

**Review by Dereck1980 — APPROVED** [review]

**Review by Dereck1980 — APPROVED** [review]

**Pull request merged**